Makefile

```
@@ -50,30 +52,81 @@ v$(VERSION).tar.$(SRC_EXT):
 $(VERSION).tar.$(SRC_EXT):
 	curl -f -L -O '$(SOURCE)'
 
+$(DEB_BUILD)/$(NAME)-$(VERSION): $(SPEC) $(SOURCES)
```

I don't think you need to set `$(SPEC)` as a prereq. here.  Also, I don't think you need to depend on `$(SOURCES)` strictly speaking, as they include the patches.  And you don't really need to depend on it being in `_topdir/SOURCES/` and can work from the copy that will be fetched to the root.

```suggestion
$(DEB_BUILD)/$(NAME)-$(VERSION): $(notdir $(SOURCE))
```

---

Changed.